src/main/resources/assets/vulkanmod/lang/ru_ru.json

```
@@ -0,0 +1,40 @@
+{
+  "vulkanmod.options.unknown": "Неизвестно",
+
+  "vulkanmod.options.pages.video": "Видео",
+  "vulkanmod.options.pages.graphics": "Графика",
+  "vulkanmod.options.pages.optimizations": "Оптимизация",
+  "vulkanmod.options.pages.other": "Другое",
+
+  "vulkanmod.options.buttons.apply": "Применить",
+  "vulkanmod.options.buttons.kofi": "Поддержите меня",
+
+  "vulkanmod.options.advCulling": "Отбраковка чанков",
+  "vulkanmod.options.advCulling.aggressive": "Агрессивная",
+  "vulkanmod.options.advCulling.conservative": "Сниженная",
+  "vulkanmod.options.advCulling.normal": "Обычная",
+  "vulkanmod.options.advCulling.tooltip": "Алгоритм отбраковкий, который может улучшить производительность благодаря уменьшению количества отображаемых чанков.",
```

У меня такой вариант был (более компактный)

```suggestion
  "vulkanmod.options.advCulling.tooltip": "Алгоритм, улучшающий производительность за счёт отсечения невидимых субчанков.",
```